pkg/kube/http_utils.go

```diff
@@ -61,36 +61,35 @@ func (c *clientWithDefaultHeaders) Do(req *http.Request) (*http.Response, error)
 	return c.client.Do(req)
 }
 
+func GetAuthHeaderFromDockerConfig(dockerConfig *credentialprovider.DockerConfigJson) (string, error) {
+	// This is a simplified handler of a Docker config which only looks for the username:password
+	// of the first entry.
```

Comment:
To avoid unknown auth errors, can we return an error if there is more than one auth as well until such a time that it's somehow possible to specify which or use all like the client does)?

Reply:
okey